changelogs/fragments/948-dellemc-migration-removal.yml

```diff
@@ -0,0 +1,9 @@
+removed_features:
+- >
+  ``ome_device_info``,  ``idrac_firmware`` and  ``idrac_server_config_profile`` modules have been removed from this collection.
```

Comment:
Can you make sure that the fragment has the exact same form as in https://github.com/ansible-collections/community.general/commit/e53f153e3071d94174036662dcba63e628a6d7a5#diff-def1d3c6591c31e8b12257385e6f6cc6155a43ec2acc8bded4a62b8fca4cbf8eR1? Thanks.

Reply:
sure

Reply:
this is done